### ISSUE TITLE
Update Dash / Zeal offline docs for Wagtail 2.9 through to 4.0

### ISSUE THREAD

**Issue by lb-** [wagtail/wagtail#9370]
### Summary

Dash & Zeal docs apps are offline documentation tools that are really helpful for locally searching documentation and also for working offline. 

We have historically added Wagtail's documentation to their 'user contributed docs' and it would be good to do an update.

### Details

- The most recent documentation in that project is v2.8.1
- https://github.com/Kapeli/Dash-User-Contributions/tree/master/docsets/Wagtail/versions
- This is not an automated process but keeping this up to date every few releases is good for the community and helps those that may use these tools for projects like Django or even the Python docs as a quick reference
- It would be good to add each version of the docs from 2.9 through to 4.0
- You can read about how to do this on the project readme - https://github.com/Kapeli/Dash-User-Contributions
- For info about these tools see https://kapeli.com/dash & https://zealdocs.org/
- When a PR is up on the external repo - add a link to it here and we can close this issue once merged into that project

#### Versions to add

Note. The patch versions can probably be ignored. 

- [ ] 2.9 https://github.com/Kapeli/Dash-User-Contributions/pull/3928 @activus-d
- [ ] 2.10 https://github.com/Kapeli/Dash-User-Contributions/pull/3929 @activus-d
- [ ] 2.11 https://github.com/Kapeli/Dash-User-Contributions/pull/3930 @activus-d
- [ ] 2.12 https://github.com/Kapeli/Dash-User-Contributions/pull/3931 @activus-d
- [ ] 2.13 https://github.com/Kapeli/Dash-User-Contributions/pull/3932 @activus-d
- [ ] 2.14 https://github.com/Kapeli/Dash-User-Contributions/pull/3936 @MaryAyobami
- [ ] 2.15 https://github.com/Kapeli/Dash-User-Contributions/pull/3937 @MaryAyobami
- [ ] 2.16 https://github.com/Kapeli/Dash-User-Contributions/pull/3938 @MaryAyobami
- [ ] 3.0 https://github.com/Kapeli/Dash-User-Contributions/pull/3939 @MaryAyobami
- [ ] 4.0 https://github.com/Kapeli/Dash-User-Contributions/pull/3940 @MaryAyobami

#### Previous updates for reference

- https://github.com/Kapeli/Dash-User-Contributions/pull/1802
- https://github.com/Kapeli/Dash-User-Contributions/pull/2758

**Comment by lb-** [comment]
Adding good first issue & Outreachy for anyone looking to get into the documentation space as it's good to understand how other projects use the docs we provide.

**Comment by zerolab** [comment]
https://hynek.me/articles/productive-fruit-fly-programmer/ has a number of interesting ideas that may be relevant

**Comment by activus-d** [comment]
@lb- @zerolab I would like to work on this.

**Comment by MaryAyobami** [comment]
@lb-  I would love to work on this too

**Comment by zerolab** [comment]
Best to divide and conquer. Please start with a version each (agree which one you're taking) and take it from there

**Comment by xomcodes** [comment]
@lb-  @zerolab  I would love to work on this too

**Comment by activus-d** [comment]
@MaryAyobami which of the versions would you like to work on?

**Comment by MaryAyobami** [comment]
@activus-d 
 If we are dividing it then I'd love to work on the last five versions

**Comment by activus-d** [comment]
@MaryAbayomi alright I would work on the earlier versions. Is "MaryAbayomi" your display name on slack? I would like to be in sync with you and also work together on some aspects that I might not understand.

**Comment by thibaudcolas** [comment]
@activus-d @MaryAyobami nice one! I’ve added your names to the list in LB’s description. @xomcodes I think two people is more reasonable, have a look at other tasks.

**Comment by dchidindu5** [comment]
Are I join the team too?

**Comment by xomcodes** [comment]
Okay @thibaudcolas . I'm still on a look out

**Comment by MaryAyobami** [comment]
> @MaryAbayomi alright I would work on the earlier versions. Is `MaryAbayomi` your display name on slack? I would like to be in sync with you and also work together on some aspects that I might not understand

@activus-d  my display name on slack is Mary Ayobami

**Comment by thibaudcolas** [comment]
@dchidindu5 @xomcodes this issue is already assigned so we should keep the discussion here to what the issue is about from now. Please reach out on Slack if you want help finding another task.

**Comment by activus-d** [comment]
@lb @thibaudcolas  

The pull requests are up. I had to make each version a separate pull request because for every version there has to be duplicate archive file in the `docsets\Wagtail` folder apart from the `specific version` folder. I don't want the file to overwrite the preceding one while trying to main the name format.

## Update Wagtail docset to version 2.9  
https://github.com/Kapeli/Dash-User-Contributions/pull/3928

## Update Wagtail to version 2.10
https://github.com/Kapeli/Dash-User-Contributions/pull/3929

## Update Wagtail to version 2.11
https://github.com/Kapeli/Dash-User-Contributions/pull/3930

## Update Wagtail to version 2.12
https://github.com/Kapeli/Dash-User-Contributions/pull/3931

## Update Wagtail to version 2.13
https://github.com/Kapeli/Dash-User-Contributions/pull/3932

**Comment by MaryAyobami** [comment]
@lb-  @thibaudcolas 
I have submitted the pull requests. I used the same method as @activus-d 

## The links 
### Wagtail version 2.14
https://github.com/Kapeli/Dash-User-Contributions/pull/3936

### Wagtail version 2.15 doc
https://github.com/Kapeli/Dash-User-Contributions/pull/3937

### Wagtail version 2.16 doc
https://github.com/Kapeli/Dash-User-Contributions/pull/3938

### Wagtail version 3.0 doc
https://github.com/Kapeli/Dash-User-Contributions/pull/3939

### Wagtail version 4.0 doc
https://github.com/Kapeli/Dash-User-Contributions/pull/3940

**Comment by lb-** [comment]
@MaryAyobami  & @activus-d - this is awesome!

Once those merge we will chat quickly on Slack about how to recognise these as a Wagtail contribution - even though it's an external package.

@thibaudcolas - idea - we get both users to submit a token PR to Wagtail with release notes updates, this way we can document this (just in release notes) and have them show up as contributors for any Outreachy records.

**Comment by lb-** [comment]
@MaryAyobami or @activus-d - it looks like the maintainer of the user docs has asked for one PR for all docs. 

Please can you coordinate this between you both. Thank you.

**Comment by activus-d** [comment]
@lb- @thibaudcolas 

## Merged Pull Request For Wagtail versions 2.9, 2.10, 2.11, 2.12
https://github.com/Kapeli/Dash-User-Contributions/pull/3944#issuecomment-1282642768

## Merged Pull Request For Wagtail version 2.13
https://github.com/Kapeli/Dash-User-Contributions/pull/3932#event-7609509180

**Comment by MaryAyobami** [comment]
@lb-  @thibaudcolas 

## Merge pull request for version 2.14,2.15,2.16,3.0 and 4.0
https://github.com/Kapeli/Dash-User-Contributions/pull/3943

**Comment by lb-** [comment]
@MaryAyobami & @activus-d - well done on working together and solving this  - you are both legends.

https://github.com/Kapeli/Dash-User-Contributions/tree/master/docsets/Wagtail/versions 

I can see all the docs are there now.

<img width="1239" alt="Screen Shot 2022-10-19 at 7 25 11 am" src="https://user-images.githubusercontent.com/1396140/196547852-4d94a47a-cd83-4292-b032-b7cf8005a460.png">

Could you both add a new PR to this main Wagtail repo with just a simple update to the 4.2 release notes.

https://github.com/wagtail/wagtail/blob/main/docs/releases/4.2.md

Inside the 'other features' section - if you can add something like...

```
* Published Wagtail docs version x.y through to x.z on https://github.com/Kapeli/Dash-User-Contributions
```

Just ping me when you have done that, but needs to be a separate PR from both of you.

I'll do the merging so we end up with one note but this way we have you confirmed as contributors for any Outreachy application.

Honestly, well done on working through this.

After this is I will raise a new one for Wagtail 4.1 release notes to go up on there also :D

**Comment by activus-d** [comment]
## Pull Request to Update Wagtail's Main Version on Dash & Zeal Docs to `Version 4.0`
https://github.com/Kapeli/Dash-User-Contributions/pull/3945

**Comment by lb-** [comment]
https://github.com/wagtail/wagtail/pull/9447

**Comment by MaryAyobami** [comment]
### Published Wagtail's doc v2.14 - v4.0 on the dash user contributions
https://github.com/wagtail/wagtail/pull/9460
